README.md

```diff
@@ -96,21 +98,24 @@ The certificate instance has the following properties:
 - `expires`: `String`. Expiration date
 - `id`: `String`. Certificate's ID
 - `isFormatValid`: `Boolean`. Indicates whether or not the certificate has a valid format
+- `issuedOn`: `String`. Datetime of issuance (ISO-8601)
 - `issuer`: `Object`. Certificate issuer
+- `metadataJson`: `Object`. Certificate metadata object
 - `name`: `String`. Name of the certificate
 - `publicKey`: `String`. Certificate's public key
-- `rawTransactionLink`: `String`. Raw transaction ID
 - `receipt`: `String`. Certificate's receipt
 - `recipientFullName`: `String`. Full name of recipient
+- `recordLink`: `String`. Link to the certificate record
 - `revocationKey`: `String|null`. Revocation key (if any)
 - `sealImage`: `String`. Raw data of the seal's image;
 - `signature`: `String`. Certificate's signature
 - `signatureImage`: `String`. Raw data of the certificate's signature image;
 - `subtitle`: `String`. Subtitle of the certificate
 - `transactionId`: `String`. Transaction ID
+- `rawTransactionLink`: `String`. Raw transaction ID
 - `transactionLink`: `String`. Transaction link
 - `verificationSteps`: `VerificationStep[]`. The array of steps the certificate will have to go through during verification
-- `version`: `CertificateVersion`. [Version of the certificate](https://github.com/blockchain-certificates/cert-verifier-js/blob/v2-wip/config/default.js#L60)
+- `version`: `CertificateVersion`. [Version of the certificate](https://github.com/blockchain-certificates/cert-verifier-js/blob/v2-wip/src/constants/certificateVersions.js)
```

Comment:
we should also add `certificateJson` to this list.

Reply:
Ah right thanks